### PR TITLE
Create masonnamminga.md

### PR DESCRIPTION
**Pull request by masonnamminga**
## Describe your project in 1-3 sentences
I am building a smart snack machine that dispenses snacks via the use of a mobile app. Users can use the web app by tapping an NFC tag, or use the built in touch screen. The project will be run off a raspberry pi, node mcu, and servo motors. 

## Hack Club Slack handle
@ Mason Namminga

## Checklist before submitting proposal:

- [x] I am a current high school student (or younger).
- [x] I have submitted an RSVP at <https://hackclub.com/winter>
- [x] I am a member of the Hack Club Slack (Join at <https://hackclub.com/slack>)
- [x] I have self-reviewed my project proposal.

### CODE REVIEW

File: masonnamminga.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+---
+name: "@masonnamminga"
+project: "Smart Snack Machine"
+---
+
+# Smart Snack Machine
+
+## Summary
+
+Here is a link to a brief sketch of the project: https://drive.google.com/file/d/1lcVTtpE1dmJsk5uPMtVKJY8mgoLhL7px/view?usp=sharing
```
Comment (bellesea) [inline masonnamminga.md:10]:
```suggestion
Here's the sketch of the project:
![Smart Snack Machine](https://user-images.githubusercontent.com/65808924/208973874-4825a188-5b4b-48c1-a1e1-f3da54840e59.png)

```

File: masonnamminga.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+## Plan
+
+I first plan to get the servos and NodeMCU's working and connected to my home network, after this I can create the webhooks that I will call in my js function to individually control the servos. 
+
+Next is assembling all the servos together and verifying that every JavaScript function is working, after all the electronics are working I will create the web app. I plan to make a simple dashboard of buttons. When a button is pressed, the dispense function will be called, allowing for the servo to rotate and drop the snack. 
+
+Finally, I will assemble the box with rows for the snacks and mount the screen and raspberry pi. For this I will use plywood and plexi glass. I will 3d print all needed brackets, and use a vinal cutter for accents and labels. The last step is to add LED light strips that can be contolled with the touchscreen!
```
Comment (bellesea) [inline masonnamminga.md:20]:
```suggestion
Finally, I will assemble the box with rows for the snacks and mount the screen and raspberry pi. For this I will use plywood and plexi glass. I will 3d print all needed brackets, and use a vinyl cutter for accents and labels. The last step is to add LED light strips that can be contolled with the touchscreen!
```

**Review by bellesea — CHANGES_REQUESTED** [review]
@masonnamminga this looks super awesome!! to check, do you have access to a 3d printer, vinyl cutter, and any other tools needed to make the body of your machine? also, would you need additional wires?

File: masonnamminga.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+| ----------------------- | ------------------------------------- | ------- |
+| Raspberry Pi 3          | Already own.                          | $0.00   |
+| Touch Screen for Pi     | https://amzn.to/3YvCXQh               | $79.99  |
+| x16 360 Degree Servos   | https://amzn.to/3YEQNzU               | $103.96 |
+| Plywood/Particle Board  | Already own.                          | $0.00   |
+| 3D Printer Filament     | Already own.                          | $0.00   |
+| x4 Node MCU WiFi Board  | https://amzn.to/3YvDPo1               | $12.59  |
+| Power Supply Module     | https://amzn.to/3YC9RhR               | $9.09   |
+| Solderless Breadboards  | https://amzn.to/3BNoFRk               | $6.69   |
+| LED WiFi Backlighting   | https://amzn.to/3PGPTi6               | $19.99  |
```
Comment (bellesea) [inline masonnamminga.md:35]: it looks like this LED light can be controlled by an external app or linked to alexa/google assistant, do you plan on linking it such that your web app can also control it? is that possible to do w/ these lights?

Comment (masonnamminga) [reply masonnamminga.md:37]: Yes, it can be. Here is a past project that uses the same idea, https://masonnamminga.com/bobby/

**Comment by masonnamminga** [comment]
> @masonnamminga this looks super awesome!! to check, do you have access to a 3d printer, vinyl cutter, and any other tools needed to make the body of your machine? also, would you need additional wires?

Yes, I do have access to a 3D printer, and vinyl cutter, as well as wires that I already own.

**Comment by bellesea** [comment]
> > @masonnamminga this looks super awesome!! to check, do you have access to a 3d printer, vinyl cutter, and any other tools needed to make the body of your machine? also, would you need additional wires?
> 
> Yes, I do have access to a 3D printer, and vinyl cutter, as well as wires that I already own.

Awesome!!

File: masonnamminga.md (hunk 3 above)
Comment (bellesea) [reply masonnamminga.md:37]: awesome!

**Review by bellesea — APPROVED** [review]
@masonnamminga this is incredible!! can't wait to see it come to life hehe :) @hackclub/bank-ops will be reaching out w/ next steps :)

**Comment by hackliv** [comment]
I'm so excited about this @masonnamminga !! This would be DANGEROUS for me though because I think I would just dispense Cheetos 24/7. So excited to see this in 10 days of making!! Dming you on Slack now about getting set up on Bank with the grant. :)

**Review by hackliv — APPROVED** [review]

**Pull request merged**